test/clientAuth-clientSecretBasic-test.js

```diff
@@ -92,4 +92,4 @@ describe('Test client secret basic', () => {
     headersDict.headers = authorizationDict;
     assert.deepEqual(headersDict, httpArgs);
   });
-});
\ No newline at end of file
+}); 
```

Comment:
Remove trailing space.

Reply:
Added trailing space for this commit as otherwise it wouldn't let me add a PR for auth0 without any changes. Was planning to review it once it was reviewed by Auth0.